testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/ManagementReadXmlTestCase.java

```diff
@@ -59,8 +61,89 @@ public class ManagementReadXmlTestCase {
     private static DomainLifecycleUtil domainPrimaryLifecycleUtil;
     private static DomainLifecycleUtil domainSecondaryLifecycleUtil;
 
+    private static void prepareExpectedServerConfiguration(String server, boolean propertyOne, int offset) throws Exception {
+        String serverConfigFileName = "expected-main-" + server + ".xml";
+        Path serverHome = new File(WildFlySecurityManager.getPropertyPrivileged("jboss.home", "toto")).toPath();
+        Path configurationDir = serverHome.resolve("standalone").resolve("configuration");
+        Files.createDirectories(configurationDir);
+        Path referenceConfiguration = configurationDir.resolve(serverConfigFileName);
+        Files.copy(new File("target/test-classes", "base-server-config.xml").toPath(), referenceConfiguration, REPLACE_EXISTING);
```

Comment:
Mmmhh, I'm not sure if it will work always. Should not we separate the path names by using the default path name separator instead of "/"?

I mean, something like:
`Files.copy(Paths.get("target").resolve("test-classes").resolve("base-server-config.xml"), referenceConfiguration, REPLACE_EXISTING);`

Reply:
oups sorry, fixing that